extensions/basic_auth/config/storage-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
nit: Is it possible to define this with a ServiceEntry?

Reply:
I tried once but service entry is not expressive enough to enforce things like V4 only (some gce only supports ipv4, but envoy resolves storage.googleapis.com to ipv6 address). Also the cluster name will be synthetic (pipe separated name) if you define the service with ServiceEntry, which is kind of weird to reference in the other Envoyfilter.